### PR TITLE
Update domains yet again (also updated node deps)

### PR DESCRIPTION
**Pull request by DoNotSpamPls**
- it-is.us
- take-me-on-a.date
- hack-my-mainfra.me
- ping-b1nzy.today

also updates depedencies and puts a webserver as a gulp task, tell me if i need to remove that, okay thanks!

### CODE REVIEW

**Comment by Proximyst** [comment]
helo yes would like to confirm is ded domen mon cause noone oosed

- jur lokel miimor gorl

**Comment by Proximyst** [comment]
hello yes @deansheather

**Comment by OkashiOdayakana** [comment]
***Wow I sure enjoy domains being removed***

**Comment by Proximyst** [comment]
daily bump !

this is a daily bump which im now starting to do cause dean is fat and itll email and annoy anyone who is subscribed to the repo until dean actually does something. thank!

**Comment by Proximyst** [comment]
very late daily bump !

this is a daily bump which im now starting to do cause dean is fat and itll email and annoy anyone who is subscribed to the repo until dean actually does something. thank!

**Comment by bramhaag** [comment]
Petition to block proxi from the issues/PRs since he's spamming my inbox 😠

File: Gulpfile.js [hunk 1]
```diff
@@ -32,4 +32,4 @@ gulp.task('pug', () => {
 gulp.task('watch', () => {
   gulp.watch('./src/scss/**/*', ['css']);
   gulp.watch('./src/views/**/*', ['pug']);
-});
+});
```
Comment (deansheather) [inline Gulpfile.js]: Where is my new line!!!!!

File: src/views/faq.pug [hunk 2]
```diff
@@ -136,9 +132,12 @@ block content
     ul
       li #[code *.are-la.me]
       li #[code *.are-pretty.sexy]
+      li #[code *.at-work.today]
       li #[code *.bad-me.me]
       li #[code *.banned.today]
       li #[code *.in-my.life]
+      li #[code *.in-this.world]
+      li #[code *.is-serious.business]
```
Comment (deansheather) [inline src/views/faq.pug]: `*.is-serious.business` is already on line 155.

**Pull request merged**